### PR TITLE
requirements-osbs.txt: remove ipython

### PR DESCRIPTION
**Pull request by flavianmissi**

### CODE REVIEW

File: requirements-osbs.txt [hunk 1]
```diff
@@ -169,8 +169,6 @@ Cython==0.29.23
 Pygments==2.9.0
```
Comment (kleesc) [inline requirements-osbs.txt:169]: I'm not sure we need this whole section; `pygments, matplotlib, ...`

Comment (flavianmissi) [reply requirements-osbs.txt]: good catch, remove the section along with some other test related deps spread throughout the file.

**Review by kleesc — APPROVED** [review]
LGTM. 
@thomasmckay Do you recall why these were included in the osbs Dockerfile in the first place?

**Pull request merged**